src/domain/participant/index.js

```diff
@@ -455,18 +457,18 @@ const adjustLimits = async (name, payload) => {
     participantExists(participant)
     let result = await ParticipantFacade.adjustLimits(participant.participantCurrencyId, limit)
     payload.name = name
-    await Utility.produceGeneralMessage(TransferEventType.NOTIFICATION, TransferEventAction.EVENT, createLimitAdjustmentMessageProtocol(payload), Utility.ENUMS.STATE.SUCCESS)
+    await Utility.produceGeneralMessage(TransferEventType.NOTIFICATION, 'limit-adjustment', createLimitAdjustmentMessageProtocol(payload), Utility.ENUMS.STATE.SUCCESS)
     return result
   } catch (err) {
     throw err
   }
 }
 
-const createLimitAdjustmentMessageProtocol = (payload, action = 'limitAdjustment', state = '', pp = '') => {
+const createLimitAdjustmentMessageProtocol = (payload, action = 'limit-adjustment', state = '', pp = '') => {
   return {
     id: Uuid(),
     from: payload.name,
-    to: 'SYSTEM',
+    to: Config.HUB_NAME,
```

Comment:
If we have config Config.HUB.NAME in central-settlement, probably it's good idea to unify it. [More info](https://github.com/mojaloop/central-settlement/pull/58).

Reply:
This is due to the use of RC module. I can delete the RC values in central-settlement and it will be the same on all places, including CEP.